chain/lf3/participation.go

```diff
@@ -92,21 +89,18 @@ func (p *Participant) run(ctx context.Context) (_err error) {
 		}
 	}()
 
+	var ticket api.F3ParticipationTicket
 	for ctx.Err() == nil {
+		var err error
 		start := time.Now()
-		ticket, err := p.tryGetF3ParticipationTicket(ctx)
+		ticket, err = p.tryGetF3ParticipationTicket(ctx, ticket)
```

Comment:
I'm _always_ passing the previous ticket in, even if we never successfully used it because we have no way to know if the failure was on our end (lotus saw the ticket and we failed to receive the lease) or on lotus's end (we failed to give them the ticket entirely).